File: src/test/java/score/StudentTest.java
package score;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    public void highestTest(){
        Student student = new Student("Ruby","048",60,100);
        Assertions.assertEquals(100,student.highest());
    }
    @Test
    public void  getAverageTest(){
        Student student = new Student("Ruby","048",60,100);
        Assertions.assertEquals((60+100)/2,student.getAverage());
    }
}

File: src/test/kotlin/scoreKt/StudentTest.kt
package scoreKt

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    val student = Student("Ruby","1024",60,100)
    @Test
    fun getAverageTest(){
        Assertions.assertEquals(((60+100)/2),student.getAverage())
    }
    @Test
    fun highestTest(){
        Assertions.assertEquals(100,student.highest())
    }
    @Test
    fun gradingTest(){
        Assertions.assertEquals('B',student.grading())
    }
    @Test
    fun passOrFailedTest(){
        Assertions.assertEquals("PASS",student.passOrFailed())
    }
}